rules/naming/src/Naming/PropertyNaming.php

```diff
@@ -44,10 +45,10 @@ final class PropertyNaming
      */
     private $inflector;
 
-    public function __construct(TypeUnwrapper $typeUnwrapper, Inflector $inflector)
+    public function __construct(TypeUnwrapper $typeUnwrapper, RectorNamingInflector $rectorNamingInflector)
     {
         $this->typeUnwrapper = $typeUnwrapper;
-        $this->inflector = $inflector;
+        $this->inflector = $rectorNamingInflector->build();
```

Comment:
@TomasVotruba, I didn't find (quickly) any better and at the same time easy way to decouple the logic for the creation of customized Inflector. If it looks way too bad from your point of you, I can invest more time into it.